sources/assets/Stride.Core.Assets/AssetImporterParameters.cs

```diff
@@ -54,6 +55,12 @@ public AssetImporterParameters(IEnumerable<Type> supportedTypes) : this()
             }
         }
 
+        /// <summary>
+        /// Gets the import input parameters.
+        /// </summary>
+        /// <value>The import input parameters.</value>
+        public PropertyCollection InputParameters { get; private set; }
```

Comment:
Small remark: no need to add `private set` anymore in recent C# versions if set only in ctor
(not a big deal, OK to keep as is -- other properties had it too anyway)